spark/src/main/java/org/opensearch/sql/spark/functions/implementation/SqlFunctionImplementation.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.functions.implementation;
+
+import static org.opensearch.sql.spark.functions.resolver.SqlTableFunctionResolver.QUERY;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.opensearch.sql.data.model.ExprValue;
+import org.opensearch.sql.data.type.ExprCoreType;
+import org.opensearch.sql.data.type.ExprType;
+import org.opensearch.sql.exception.ExpressionEvaluationException;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.expression.FunctionExpression;
+import org.opensearch.sql.expression.NamedArgumentExpression;
+import org.opensearch.sql.expression.env.Environment;
+import org.opensearch.sql.expression.function.FunctionName;
+import org.opensearch.sql.expression.function.TableFunctionImplementation;
+import org.opensearch.sql.spark.client.SparkClient;
+import org.opensearch.sql.spark.request.SparkQueryRequest;
+import org.opensearch.sql.spark.storage.SparkMetricTable;
+import org.opensearch.sql.storage.Table;
+
+/**
+ * Spark SQL function implementation.
+ */
+public class SqlFunctionImplementation extends FunctionExpression
```

Comment:
np: maybe add Spark prefix, `SparkSqlxxx`. same for SqlTableFunctionResolver

Reply:
Thanks! Added prefix.